ui/desktop/electron-app/src/services/client-daemon-manager.js

```diff
@@ -65,11 +72,16 @@ class ClientDaemonManager {
    * @param tokenId
    * @returns {Promise}
    */
-  addToken({ token, tokenId }) {
+  async addToken({ token, tokenId }) {
+    // TODO: Should this and search just always call CLI even for non-Windows?
+    if (isWindows()) {
+      return addTokenCliCommand(token);
+    }
+
     const postBody = {
-      auth_token_id: tokenId,
+      auth_token_id: sanitizer.base62EscapeAndValidate(tokenId),
```

Comment:
why are we sanitizing the token? don't we always send the tokenId as is to the backend?

Reply:
While reviewing some of the code for the `boundary connect` command, we noticed that there was a sanitize helper for any info going in, so we decided to match that here as well. Mainly to be consistent.